src/templates/search.jsx

```diff
@@ -1,8 +1,9 @@
+
 // import {graphql} from 'gatsby';
 import React from 'react';
 import querystring from 'querystring';
 import PropTypes from 'prop-types';
-import {navigate} from 'gatsby';
+import {navigate,useStaticQuery, graphql} from 'gatsby';
```

Comment:
minor nit about consistent spacing. I'm actually surprised there's no linting rule enabled for it

can you change it to either
```suggestion
 import {navigate,useStaticQuery,graphql} from 'gatsby';
```
or
```suggestion
 import {navigate, useStaticQuery, graphql} from 'gatsby';
```